projects/gnome.org/json-glib/package.yml

```diff
@@ -0,0 +1,43 @@
+distributable:
+  url: https://download.gnome.org/sources/json-glib/{{version.marketing}}/json-glib-{{version}}.tar.xz
+  strip-components: 1
+versions:
+  url: https://download.gnome.org/sources/json-glib/cache.json
+  match: /json-glib-\d+\.\d+\.\d+\.tar\.xz/
+  strip:
+    - /^json-glib-/
+    - /\.tar\.xz/
+dependencies:
+  gnome.org/glib: '*'
```

Comment:
dangerous to depend on `*`, we recently saw breakage due to this with postgres and unicode.org